### ISSUE TITLE
Reproducibility

### ISSUE THREAD

**Issue by sharpobject** [GitSparTV/LuaJIT-Benchmarks#2]
Hello,

It would be sweet if people could see the script used to generate the results and run that script themselves. Manually editing the benchmark script is a slow and error-prone process.

**Comment by GitSparTV** [comment]
Yes I was thinking about this but I need to find the way when the conveuor of tests doesn't effect the tests themselves. Since I planned to re run all tests (yes again) this may go to the next update

**Comment by GitSparTV** [comment]
@sharpobject Hi, I'm starting to working on this feature if you have any ideas how can I make generation better, feedback is welcome
